src/utils/dig.js

```diff
@@ -28,15 +28,15 @@ export const validateDomain = (input, response) => {
     };
 };
 
-export const handleDig = async ({ domain, types, short, provider }) => {
+export const handleDig = async ({ domain, types, short, cdflag ,provider }) => {
     // Make the DNS queries
     const results = await Promise.all(types.map(type =>
         performLookupWithCache(domain, type, provider.doh).then(data => ({ type, data }))));
```

Comment:
You need to pass the cdflag into the actual DoH call, so that we can actually disable it -- everything you've done so far is just visual.

Reply:
OOPS my bad. I have a lot to learn from this haha

Reply:
Hey @MattIPv4 so, I need to add one more parameter, `cdflag` to `performLookupWithCache` and `performLookup` right?

Reply:
Yup, and then pass it into the DoH call itself.

https://github.com/MattIPv4/DNS-over-Discord/blob/8bbb3f6a5351e6ce22ec56d4e1449686fd71c918/src/utils/dns.js#L94-L103

https://github.com/MattIPv4/DNS-over-Discord/blob/8bbb3f6a5351e6ce22ec56d4e1449686fd71c918/src/utils/dns.js#L78-L81